src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,25 @@
+@use "../utils/variables" as *;
+
+.stars {
+  display: flex;
+  width: 166px;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+
+  &__star {
+    width: $Size-casual;
+    height: $Size-casual;
+    flex-shrink: 0;
+    position: relative;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  .stars__star:not(:nth-child(n+5)) {
+    background-image: url(../images/star-active.svg);
```

Comment:
You are still not using the `stars--N` modifier. Same as in the stars layout, it should be a second class of the stars wrapper. Basing on that, you will be able to highlight first N stars.
Please refer to your stars layout solution